tests/job_tests/returnn/test_search.py

```diff
@@ -0,0 +1,37 @@
+import tempfile
+from sisyphus import setup_path
+
+from i6_core.returnn.search import SearchBPEtoWordsJob
+from i6_core.util import check_file_sha256_checksum, compute_file_sha256_checksum
+
+Path = setup_path(__package__)
+
+
+def test_search_bpe_to_words_single():
+    search_out = Path("files/search_out_single")
+    reference_word_search_results = Path("files/word_search_results_single.py")
+    bpe_to_words_job = SearchBPEtoWordsJob(search_out)
+    bpe_to_words_job.out_word_search_results = Path(tempfile.mktemp(".py"))
```

Comment:
Why is this necessary?

Reply:
En passant, `tempfile.mktemp` is deprecated https://docs.python.org/3/library/tempfile.html#deprecated-functions-and-variables

Reply:
Good to know, I thought there would be warning in PyCharm if you use deprecated methods.

Reply:
> Can it be possible to add some settings in sisyphus to change the target directory to a temporary one for test purpose?

I am not sure how simple this is, you can have a look at it if you want.